xilem_core/src/views/one_of.rs

```diff
@@ -84,6 +84,33 @@ where
     }
 }
 
+impl<T, A, B, C, D, E, F, G, H, I> AsMut<T> for OneOf<A, B, C, D, E, F, G, H, I>
+where
+    A: AsMut<T>,
+    B: AsMut<T>,
+    C: AsMut<T>,
+    D: AsMut<T>,
+    E: AsMut<T>,
+    F: AsMut<T>,
+    G: AsMut<T>,
+    H: AsMut<T>,
+    I: AsMut<T>,
+{
+    fn as_mut(&mut self) -> &mut T {
+        match self {
+            OneOf::A(e) => <A as AsMut<T>>::as_mut(e),
+            OneOf::B(e) => <B as AsMut<T>>::as_mut(e),
+            OneOf::C(e) => <C as AsMut<T>>::as_mut(e),
+            OneOf::D(e) => <D as AsMut<T>>::as_mut(e),
+            OneOf::E(e) => <E as AsMut<T>>::as_mut(e),
+            OneOf::F(e) => <F as AsMut<T>>::as_mut(e),
+            OneOf::G(e) => <G as AsMut<T>>::as_mut(e),
+            OneOf::H(e) => <H as AsMut<T>>::as_mut(e),
+            OneOf::I(e) => <I as AsMut<T>>::as_mut(e),
+        }
+    }
+}
+
```

Comment:
This resulted from some experiments prior to the `With` trait, but I don't think it hurts to include this as well to xilem_core.

Reply:
I don't *really* like the existing `AsRef` implementation, but I've not spent too much time reasoning about it. That being said, this is fine to add this for consistency.